# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../dietlibc/ppc64-atexit.patch
# Copyright (C) 2015 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- dietlibc-0.33/ppc64/start.S.vanilla	2015-10-28 18:01:19.194858227 +0000
+++ dietlibc-0.33/ppc64/start.S	2015-10-28 18:00:21.355000409 +0000
@@ -67,7 +67,7 @@
 #define DOTIFY(name) .##name
 	bl	DOTIFY(CALL_IN_STARTCODE)
 #endif
-	b	.exit
+	b	.__libc_exit
 
 .LT_start:
 	.long	0
